### PR TITLE
fix punishments nomenclature for clarity

### PR DESCRIPTION
**Pull request by ines-toupeira**
This update clarifies the distinction between penalties and jailing under the term "punishments". Penalties now refer to lighter misbehaviors, such as delaying block production, while jailing applies to more severe misbehaviors, like attempting to fork the chain. The term 'penalty' was previously used ambiguously to describe both block delay penalties and jailing

### CODE REVIEW

File: learn/protocol/glossary.md [hunk 1]
```diff
@@ -98,7 +98,7 @@ An inherent is a type of data that is intrinsic to the block. Unlike transaction
 
 ## Jail
 
-Going to jail refers to the validator's state characterized by incurring severe penalties for actions such as forking or continuing on a fork. When a validator is in a jailed state, it is locked for an 8-epoch period and cannot be re-elected during this lockup. Additionally, all the validator’s rewards are burned.
+Going to jail refers to the validator's state characterized by incurring severe misbehaviors for actions such as forking or continuing on a fork. When a validator is in a jailed state, it is locked for an 8-epoch period and cannot be re-elected during this lockup. Additionally, all the validator’s rewards are burned.
```
Comment (ii-cruz) [inline learn/protocol/glossary.md]:
```suggestion
Going to jail refers to the validator's state characterized by incurring severe misbehavior for actions such as forking or continuing on a fork. When a validator is in a jailed state, it is locked for an 8-epoch period and cannot be re-elected during this lockup. Additionally, all the validator’s rewards are burned.
```

File: learn/protocol/fork-proofs.md
Comment (ii-cruz) [inline learn/protocol/fork-proofs.md:1]: idk if this was intentional. Is the fork proofs doc supposed to go away?

**Review by ii-cruz — COMMENTED** [review]
Other than what I commented it is good to go :)

File: learn/protocol/fork-proofs.md
Comment (ines-toupeira) [reply learn/protocol/fork-proofs.md:1]: yes! I included the fork proofs in the equivocation proofs docs

**Review by ii-cruz — APPROVED** [review]

**Pull request merged**